magefile.go

```diff
@@ -33,7 +33,7 @@ var (
 	publicDir      = "./public"
 	buildDir       = "./build"
 	storageDir     = "./dev/packages/storage"
-	storageRepoDir = "./.package-storage"
+	storageRepoDir = filepath.Join(buildDir, ".package-storage")
```

Comment:
Lets not make it a `.`. Otherwise browsing it is harder.

Reply:
Fixed.